packages/keystone/KSv5v6/v5/prepareKeystone.js

```diff
@@ -101,6 +101,14 @@ function prepareKeystone ({ onConnect, extendKeystoneConfig, extendExpressApp, s
         ...extendedKeystoneConfig,
     })
 
+    // patch access control handler to store skipAccessControl flag in context
+    const accessControlContextHandler = keystone._getAccessControlContext
+    keystone._getAccessControlContext = args => {
```

Comment:
it looks like you want to patch `createContext`! It's a bad idea to patch a private method